xdsl/dialects/__init__.py

```diff
@@ -0,0 +1,310 @@
+from collections.abc import Callable
```

Comment:
Oh, I would not have thought about adding this to the `__init__.py` directly, but that's a really good idea!